src/App/Fossa/LicenseScanner.hs

```diff
@@ -216,10 +213,10 @@ uploadVendoredDep apiOpts VendoredDependency{..} themisScanResult = do
     Nothing -> pure . showT . md5 . encodeUtf8 . show . NE.sort $ themisScanResult
     Just version -> pure version
 
-  signedURL <- Fossa.getSignedLicenseScanURL apiOpts depVersion vendoredName
+  signedURL <- getSignedLicenseScanUrl $ PackageRevision{packageVersion = depVersion, packageName = vendoredName}
```

Comment:
[optional] [nit] why not `PackageRevision vendoredName depVersion` like you have in the Archive uploader?

Reply:
I actually think they should both be this way.  The point is to name these `Text` parameters that are being passed in so it's hard to accidentally get them switched around.